components/kyma-environment-broker/common/orchestration/strategies/parallel.go

```diff
@@ -0,0 +1,174 @@
+package strategies
```

Comment:
diff in /strategies files is misleading because I moved them to a separate package. I've changed mostly processOperation function. To see the difference, look at the deleted file.